src/engine/specification.md

```diff
@@ -197,25 +223,35 @@ This structure contains the attributes required to initiate a payload build proc
 #### Response
 
 * result: `object`
-    - `status`: `enum` - `"SUCCESS" | "SYNCING"`
+    - `status`: `enum` - `"VALID" | "INVALID" | "SYNCING" | "ACCEPTED"`
```

Comment:
Ditto on from `newPayload`. I suggest returning a `payloadStatus` object along with `payloadId` for better encapsulation. And specific which `status`s can be returnd here from a global set (i.e. the full set less `INVALID_BLOCK_HASH`)